contents/docs/surveys/adding-surveys-code.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Adding survey code
```

Comment:
This is slightly confusing because you need to "add code" for the popover surveys (just the snippet or library still counts as code.

Maybe "Implementing custom surveys"?